### ISSUE TITLE
timeGrid events incorrectly overlapping

### ISSUE THREAD

**Issue by arshaw** [fullcalendar/fullcalendar#2758]
Originally reported on Google Code with ID 2493

```

I found an unexpected scenario on fullcalendar where events start hiding each other
when many are on the screen. The minimal set I found to reproduce this error is composed
by nothing less than 12 events, and if any of these 12 events is removed or changed,
the bug disappears.

Please take a look on the demo at http://jsbin.com/xizujo/1/edit?js,output (or http://plnkr.co/F2bzehY8Pc9Vo3D4H5Lc)
and search for the events L and B. You will actually not be able to find B because
L is hiding it, so move L far from its position and B is gonna appear. The screenshots
1 and 2 show this process.

Does anybody has any clue what is going on with event B?
```

Reported by `vd@churchdesk.com` on 2015-04-28 10:01:28

<hr>
- _Attachment: 1.png<br>![1.png](https://storage.googleapis.com/google-code-attachments/fullcalendar/issue-2493/comment-0/1.png)_
- _Attachment: 2.png<br>![2.png](https://storage.googleapis.com/google-code-attachments/fullcalendar/issue-2493/comment-0/2.png)_

Imported with **2** stars.

**Comment by arshaw** [comment]
Depiction of `vd@churchdesk.com`'s JSBin from a while ago:
![screen shot 2017-04-27 at 6 20 50 pm](https://cloud.githubusercontent.com/assets/56520/25506776/4f400e78-2b76-11e7-8755-6475fc7de6db.png)
http://jsbin.com/xizujo/2/

**Comment by arshaw** [comment]
I'm not sure when this was fixed, but it's fixed now (as of v5.5.1)

https://codepen.io/arshaw/pen/ExZKNze?editors=0010
(notice how event "B" is now visible)
